src/components/PhotoSwipe/PhotoSwipe.tsx

```diff
@@ -839,7 +837,17 @@ function PhotoSwipe(props: Iprops) {
                 ref={pswpElement}>
                 <div className="pswp__bg" />
                 <div className="pswp__scroll-wrap">
-                    {livePhotoBtn}
+                    <LivePhotoBtn
+                        onClick={livePhotoBtnOptions.click}
+                        onMouseEnter={livePhotoBtnOptions.show}
+                        onMouseLeave={livePhotoBtnOptions.hide}
+                        style={{
+                            display: livePhotoBtnOptions.visible
+                                ? 'block'
+                                : 'none',
+                        }}>
+                        {livePhotoBtnHTML} LIVE
```

Comment:
we should shift this text to `englishConstants` File

context -> we keep all the constants in the `englishConstants` file. So that in future we can support localisation with minimal code changes.